src/pages/ItemsList/CheckButton/CheckBox.js

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const CheckBox = ({ id, name, checkCheckBox, themeData, handlerThemeInfo }) => {
+  const [saveThemeId, setSaveCheckThemeId] = useState(id);
+
+  const saveCheckTheme = ({ target }) => {
+    setSaveCheckThemeId(target.id);
+    handlerThemeInfo(saveThemeId, target.name);
+  };
+
+  return (
+    <CheckBoxContainer htmlFor={id}>
+      <CheckBoxInput
+        name={name}
+        id={id}
+        checked={themeData.includes(String(id)) ? true : false}
+        onChange={e => checkCheckBox(e)}
```

Comment:
이벤트가 인자로 전달되기 때문에 아래와 같이 작성할 수 있습니다.
```suggestion
        onChange={checkCheckBox}
```